scripts/hud/cgaz.js

```diff
@@ -718,17 +726,19 @@ class Cgaz {
 
 	static getNormal(vec, threshold) {
 		const mag = this.getSize(vec);
+		let vecNormal = {
```

Comment:
This can be `const`, the properties in a JS object are still mutable. Weird I know, but good practice